docs/src/content/docs/id/reference/frontmatter.md

```diff
@@ -0,0 +1,353 @@
+    file: ../../assets/logo.png
+  actions:
+    - text: Ceritakan lebih banyak
+      link: /getting-started/
+      icon: right-arrow
+      variant: primary
+    - text: Lihat di GitHub
+      link: https://github.com/astronaut/my-project
+      icon: external
+---
+```
+
+Anda dapat menampilkan versi gambar _hero_ yang berbeda dalam mode terang dan gelap.
+
+```md
+---
+hero:
+  image:
+    alt: Logo yang berkilauan dan berwarna cerah
+    dark: ../../assets/logo-dark.png
+    light: ../../assets/logo-light.png
+---
+```
+
+#### `HeroConfig`
+
+```ts
+interface HeroConfig {
+  title?: string;
+  tagline?: string;
+  image?:
+    | {
+        // Path relatif ke gambar di repositori Anda.
+        file: string;
+        // Teks alternatif untuk membuat gambar dapat diakses oleh teknologi bantu
+        alt?: string;
+      }
+    | {
+        // Path relatif ke gambar di repositori Anda yang akan digunakan untuk mode gelap.
+        dark: string;
+        // Path relatif ke gambar di repositori Anda yang akan digunakan untuk mode terang.
+        light: string;
+        // Teks alternatif untuk membuat gambar dapat diakses oleh teknologi bantu
+        alt?: string;
+      }
+    | {
+        // HTML mentah (raw) untuk digunakan dalam slot gambar.
+        // Bisa berupa tag `<img>` khusus atau tag `<svg>` sebaris.
+        html: string;
+      };
+  actions?: Array<{
+    text: string;
+    link: string;
+    variant: 'primary' | 'secondary' | 'minimal';
+    icon: string;
+  }>;
+}
+```
+
+### `banner`
+
+**type:** `{ content: string }`
+
+Menampilkan _banner_  pengumuman di bagian atas halaman ini.
+
+Nilai `content` dapat berupa HTML untuk tautan atau konten lainnya.
+Sebagai contoh, halaman ini menampilkan spanduk yang menyertakan tautan ke `example.com`.
+
+```md
+---
+title: Halaman dengan banner
+banner:
+  content: |
+    Kami baru saja meluncurkan sesuatu yang keren!
+    <a href="https://example.com">Lihatlah</a>
+---
+```
+
+### `lastUpdated`
+
+**type:** `Date | boolean`
+
+Mengesampingkan [opsi `lastUpdated` global](/id/reference/configuration/#lastupdated). Jika tanggal ditentukan, tanggal tersebut harus berupa [stempel waktu YAML](https://yaml.org/type/timestamp.html) yang valid dan akan menimpa tanggal yang tersimpan dalam riwayat Git untuk halaman ini.
+
+```md
+---
+title: Halaman dengan tanggal pembaruan terakhir khusus
+lastUpdated: 2022-08-09
+---
+```
+
+### `prev`
+
+**type:** `boolean | string | { link?: string; label?: string }`
+
+Mengesampingkan [opsi `pagination` global](/id/reference/configuration/#pagination). Jika string ditentukan, teks tautan yang dihasilkan akan diganti dan jika objek ditentukan, tautan dan teks akan ditimpa.
+
+```md
+---
+# Menyembunyikan tautan halaman sebelumnya
+prev: false
+---
+```
+
+```md
+---
+# Mengganti teks tautan halaman sebelumnya
+prev: lanjutkan tutorial
+---
+```
+
+```md
+---
+# Mengabaikan tautan dan teks halaman sebelumnya
+prev:
+  link: /unrelated-page/
```

Comment:
Not sure if needed, but the link can use translated keywords as Indonesian words may be easier to understand this is a totally unrelated page. Any thoughts?

Reply:
You're right, maybe I should translate it